listenbrainz/webserver/views/playlist_api.py

```diff
@@ -923,7 +938,7 @@ def import_playlist_from_spotify(service):
 @crossdomain
 @ratelimit()
 @api_listenstore_needed
-def import_tracks_from_spotify_to_playlist(service, playlist_id):
+def import_tracks_from_spotify_playlist(service, playlist_id):
```

Comment:
Thanks for the feedback! I think for now I will leave them as separate endpoints, and remove the service arg. I will open a separate PR to handle that this later, since for now I am not yet sure how to better scale it as more music services will be added.

Reply:
Great. My only advice would be to note that down on a list somewhere, because I know for sure I will forget myself :p